### ISSUE TITLE
Batch adding repos - misses some out

### ISSUE THREAD

**Issue by rdc74656** [wstyres/Zebra#1427]
I recently copied sources from a flame export I emailed myself and when I copied and batch added into zebra, it was only once I was searching for a package that I found out that zebra had not installed that repo.

**To Reproduce**
Copy an extensive list of repos to clipboard and use zebra batch add option. Then just double check once sources refreshed.

**Expected behavior**
Any and all sources to be added to zebra when using batch add feature with any issues shown once sources refreshed.

iOS Version [e.g. 12.1.2 or 10.3.3]: 13.3
 - Zebra Version [e.g. 1.0\~beta1 or beta1]: 1.1.10
 - Device Type [e.g. iPhone or iPad]: iPhone 11 Pro
 - Jailbreak Type [e.g. unc0ver, Chimera, checkra1n, etc.]: unc0ver 5.2.0

**Comment by mwoolweaver** [comment]
would you mind posting the list?

**Comment by rdc74656** [comment]
[ZGE.txt](https://github.com/wstyres/Zebra/files/4801015/ZGE.txt)

**Comment by wstyres** [comment]
Duplicate of #1290
